app/utils/notifications.ts

```diff
@@ -33,10 +35,20 @@ export const addDeviceToken = async (client: ApolloClient<unknown>): Promise<voi
 }
 
 export const hasNotificationPermission = async (): Promise<boolean> => {
-  const authorizationStatus = await messaging().hasPermission()
+  if (Platform.OS === "ios") {
+    const authorizationStatus = await messaging().hasPermission()
+    return (
+      authorizationStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+      authorizationStatus === messaging.AuthorizationStatus.PROVISIONAL
+    )
+  }
+
+  if (Platform.OS === "android") {
+    const authorizationStatusAndroid = await PermissionsAndroid.request(
+      PermissionsAndroid.PERMISSIONS.POST_NOTIFICATIONS,
+    )
+    return authorizationStatusAndroid === PermissionsAndroid.RESULTS.GRANTED || false
+  }
```

Comment:
Need this android permission to work on my Pixel 6 (and newer version of android) see https://rnfirebase.io/messaging/usage#android---requesting-permissions